src/StackExchange.Redis/Interfaces/IDatabase.cs

```diff
@@ -1102,6 +1102,19 @@ public interface IDatabase : IRedis, IDatabaseAsync
         /// <remarks>https://redis.io/commands/sismember</remarks>
         bool SetContains(RedisKey key, RedisValue value, CommandFlags flags = CommandFlags.None);
 
+        /// <summary>
+        /// Returns whether each member is a member of the set stored at key.
+        /// </summary>
+        /// <param name="key">The key of the set.</param>
+        /// <param name="values">The members to check for.</param>
+        /// <param name="flags">The flags to use for this operation.</param>
+        /// <returns>
+        /// <see langword="true"/> if the element is a member of the set.
+        /// <see langword="false"/> if the element is not a member of the set, or if key does not exist.
+        /// </returns>
```

Comment:
In this case doc should say an _array_ of these true/false, matching position, e.g. 
```suggestion
        /// <returns>
        /// An array of booleans corresponding to <paramref name="values"/>, for each:
        /// <see langword="true"/> if the element is a member of the set.
        /// <see langword="false"/> if the element is not a member of the set, or if key does not exist.
        /// </returns>
```

What happens when the key isn't present, all falses?

Reply:
yes